src/lib/index.ts

```diff
@@ -2,5 +2,5 @@ import { Client } from "./client";
 
 export default new Client();
 
-export * from "./client";
+export type * from "./client";
```

Comment:
This was previously exporting `Client` as a class, this changes it to only export the type of `Client` so we only have a single default export.